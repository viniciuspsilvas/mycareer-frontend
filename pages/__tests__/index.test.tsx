import { render } from '@testing-library/react'
import Home from '../index'

jest.mock(
  '@components/sections/AwardSection',
  () =>
    function AwardSection() {
      return <div>I am AwardSection</div>
    }
)

describe('Home', () => {
  test('renders Home unchanged', () => {
    const { container } = render(<Home />)
    expect(container).toMatchSnapshot()
  })
})
