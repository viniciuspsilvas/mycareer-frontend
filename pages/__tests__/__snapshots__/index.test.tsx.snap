// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders Home unchanged 1`] = `
<div>
  <div>
    <div>
      I am AwardSection
    </div>
  </div>
</div>
`;
